asab/web/tenant/midleware.py

```diff
@@ -4,7 +4,15 @@
 import asab.web.rest
 
 
-def tenant_middleware_factory(app, svc):
+def tenant_middleware_factory(app, svc, trusted=False):
```

Comment:
`trusted` is a configuration item, taken from `TenantService`

It means that if you receive an unknown tenant, you will *add* the tenant into a list of tenants (so it will become known).